mapper-pojo/src/main/java/org/hibernate/search/v6poc/entity/pojo/processing/building/impl/PojoIndexingProcessorValueNodeBuilderDelegate.java

```diff
@@ -93,7 +92,7 @@ public void indexedEmbedded(String relativePrefix, ObjectFieldStorage storage,
 			PojoIndexingProcessorTypeNodeBuilder<V> nestedProcessorBuilder = new PojoIndexingProcessorTypeNodeBuilder<>(
 					embeddedTypeModelPath, mappingHelper, nestedBindingContext,
 					// Do NOT propagate the identity mapping collector to IndexedEmbeddeds
-					PojoIdentityMappingCollector.noOp()
+					null
```

Comment:
Just so that we end up being on the same page: considering our latest discussions and considering it's purely internal, I would have used an `Optional` here. But apparently, that's not how you see things?

Reply:
A good article that sums up the problem: https://dzone.com/articles/optional-method-parameters

I'm a bit afraid it'll come back to bite us, but ok, considering it's internal, and bootstrap code, I'll change that. And I'll try to keep that in mind in the future.

Reply:
Done.